store/src/main/java/com/denis/store/utility/DBInitializer.java

```diff
@@ -0,0 +1,47 @@
+package com.denis.store.utility;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Objects;
+
+public class DBInitializer {
+
+    public static void init() {
+        Connection connection = null;
+
+        try {
+            connection = ConnectionPool.getConnection();
+            Statement statement = connection.createStatement();
+
+            try {
+                statement.executeUpdate("DROP TABLE Products");
+                statement.executeUpdate("DROP TABLE Category");
```

Comment:
also, when you create on (DBPopulator) and invoke this method, your DB will be empty, from where do you planned to get products? If you have some misunderstanding about task requirements - feel free to contact me